manifests/application.pp

```diff
@@ -21,7 +21,7 @@
 class katello::application (
   Integer[0] $rest_client_timeout = 3600,
   Optional[Enum['SSLv23', 'TLSv1', '']] $cdn_ssl_version = undef,
-  Optional[Stdlib::Host] $proxy_host = undef,
+  Optional[Variant[Stdlib::Host, String]] $proxy_host = undef,
```

Comment:
So previously this was a `Stdlib::Httpurl` and it may have been my mistake that I didn't change it. I don't know if Katello actually accepts a plain host. Probably easier to revert back to that.
```suggestion
  Optional[Stdlib::Httpurl] $proxy_host = undef,
```